src/Robo/Plugin/Commands/ThemeCommands.php

```diff
@@ -35,8 +35,13 @@ public function __construct()
      */
     public function themeBuild($siteName = 'default'): Result
     {
-        $themeBuildConfiguration = $this->getConfig('theme_build', $siteName);
-        $this->io()->title("theme build");
+        $result = $this->io()->title("theme build");
+        try {
+            $themeBuildConfiguration = $this->getConfig('theme_build', $siteName);
+        } catch (TaskException $e) {
+            $this->say("'$siteName' theme_build confguration not set.");
+            return $this->taskExec('echo skipping')->run();
```

Comment:
Will this just print `skipping` then? Is there a more robo way to do this?

Reply:
The problem is I need to return a `Result` object from this method (well, I would _like_ to) and if I bail because I don't get a theme configuration back, then we weren't actually doing anything robo-wise that returns a `Result` object.

Reply:
Could say this the robo way, then just create a `Result(TRUE)` object, or however you would do that?

Reply:
It's a little more complicated than that: https://github.com/consolidation/Robo/blob/063a9db302b01d1a7ffa52a8823fceb48312fc9b/src/Result.php#L21-L36

Reply:
Ya, I figured my idea might be too good to be true, but was hoping it sparked an idea for you. 🤷🏼